### DISCUSSION TITLE
Nuget package for Serilog.Exceptions.SqlServer is missing

### DISCUSSION THREAD

**Discussion by cd21h** [RehanSaeed/Serilog.Exceptions#63] [Q&A]
Hi,
is nuget for SqlServer destructurer published?

**Comment by RehanSaeed** [comment]
Not published. It contains a single solitary class at the moment. I think the idea of having separate assemblies to add extra destructurers is a good one. We just need to put some more effort in to add more of them.

**Comment by RehanSaeed** [comment]
Package is now available:

https://www.nuget.org/packages/Serilog.Exceptions.SqlServer/
